ZappiOS/Podfile

```
@@ -1,5 +1,9 @@
 # Uncomment the next line to define a global platform for your project
+load 'Flipper'
+
 platform :ios, '11.0'
+require_relative './node_modules/@react-native-community/cli-platform-ios/native_modules'
+require_relative '../scripts/autolink.rb'
```

```suggestion
require_relative '../scripts/autolink-react-dependencies.rb'
```

this doesn't match the name of the script file you added above
are there scenarios where we might run this outside of this repo ? in which case it could be in an npm package, no ?

---

updated